assets/javascripts/discourse/components/assigned-to-post.gjs

```diff
@@ -0,0 +1,64 @@
+import Component from "@glimmer/component";
+import { action } from "@ember/object";
+import { inject as service } from "@ember/service";
+import DButton from "discourse/components/d-button";
+import icon from "discourse-common/helpers/d-icon";
+import i18n from "discourse-common/helpers/i18n";
+import DMenu from "float-kit/components/d-menu";
+
+export default class AssignedToPost extends Component {
+  @service taskActions;
+
+  @action
+  unassign() {
+    this.taskActions.unassignPost(this.args.post);
+  }
+
+  @action
+  editAssignment() {
+    this.taskActions.showAssignPostModal(this.args.post);
+  }
+
+  <template>
+    {{#if @assignedToUser}}
+      {{icon "user-plus"}}
+    {{else}}
+      {{icon "group-plus"}}
+    {{/if}}
+
+    <span class="assign-text">
+      {{i18n "discourse_assign.assigned_to"}}
+    </span>
+
+    <a href={{@href}} class="assigned-to-username">
+      {{#if @assignedToUser}}
+        {{@assignedToUser.username}}
+      {{else}}
+        {{@assignedToGroup.name}}
+      {{/if}}
+    </a>
+
+    <DMenu @icon="ellipsis-h" class="btn-flat more-button">
+      <div class="popup-menu">
```

Comment:
is this necessary ?

Reply:
Yes, we need that to style the popup menu appropriately.

Reply:
(`DMenu` doesn't provide that)